### ISSUE TITLE
Pressing the left arrow causes an error

### ISSUE THREAD

**Issue by jdavis** [aseprite/aseprite#1591]
When using the Rectangular Marquee Tool and when I hit the left arrow, it will show a popup with an error. Yet all the other arrows are fine...

### Aseprite and System version

* Aseprite version: 1.2.3 (just upgraded)
* System: macOS 10.12.6

Video of the error: https://twitter.com/hopefuljosh/status/919414608577753091

Uploaded to Twitter since GitHub doesn't allow videos? Also I'm going to have to downgrade because this is pretty blocking.

**Comment by dacap** [comment]
I cannot reproduce it yet, but I think it might be related to the data recovery feature that is trying to save the background information while we move the selection. I'll be checking this one.

**Comment by jdavis** [comment]
I just upgraded to 1.2.6 in the hopes that it was fixed... I still see this error.

Anything I can do to help debug it more? It makes using Aseprite very difficult.

**Comment by jdavis** [comment]
Here's another video of another part of the bug that I found: https://twitter.com/jldavis/status/947691193835470848

It makes me wonder if it has to do with the autoselect layer?? Do you see how the lines disappear behind where the autoselect blue lines were?

**Comment by jdavis** [comment]
More strangeness https://twitter.com/jldavis/status/947738765421789185

**Comment by jdavis** [comment]
When you said it wasn't reproducible, it got me thinking that it had to be something I did.

I did the classic rename the *Application Support* on macOS and then did a process of elimination to figure out what the issue was.

I figured out that it was the `user.aseprite-keys` file so obviously I messed up some keybindings. I then looked at the left key and this was the difference between this and a clean install of Aseprite:

```xml
        <key command="MoveMask" shortcut="Left">
            <param name="direction" value="left" />
            <param name="quantity" value="1" />
            <param name="target" value="content" />
            <param name="units" value="pixel" />
            <param name="wrap" value="1" />
        </key>
```

I changed that and there's no longer an error.

Thanks and sorry for bothering you!
